src/components/helpers/helpers.js

```diff
@@ -15,3 +16,20 @@ export const getUsername = token => {
     }
     return null;
 };
+
+export const formatDate = eventDate => {
```

Comment:
What do you think about using `toDateString()` date function instead of doing your own formatting

Reply:
using `toDateString()` will output something like `Mon May 21 2018`  which does not conform to the date field in the database model i.e `db.DateTime` hence need to manually convert it to `YYYY-MM-DD` format.

Reply:
@Rodgers-M you could use [human-date](https://www.npmjs.com/package/human-date)